uptane-standard.md

```diff
@@ -701,7 +706,7 @@ An ECU version report is a metadata structure that SHALL contain the following i
 
 #### Download and check current time {#check_time_primary}
 
-The Primary SHALL load the current time from a secure source.
+The Primary SHALL load and verify the current time from a secure source.
```

Comment:
We need to discuss this in the meeting.  I think the word verified may be intentionally omitted because we don't know about the verification process and it may not be possible to do something meaningful in all cases.

Reply:
In a later section "Full verification" step 1, it explicitly says "Load and verify the current time or the most recent securely attested time."

This is what both the Primary and Secondary ECUs are performing. So I thought it should state "verify" in this section as well. Also the title is "Download and check current time" but only the "download" part is mentioned

Reply:
Okay, this makes sense to me.  Let us bring this up in the meeting to see if anyone else objects.  This isn't a material change (in my eyes), so can fit under a 0.0.X spec change.

Reply:
After discussing this at the 5/9 Uptane Standards meeting, the consensus was to not add the suggested reference to verifying time in line 709.